packages/service-library/src/servicelib/utils_secrets.py

```diff
@@ -57,3 +57,39 @@ def secure_randint(start: StrictInt, end: StrictInt) -> int:
 
     diff = end - start
     return secrets.randbelow(diff) + start
+
+
+_PLACEHOLDER: Final[str] = "*" * 8
```

Comment:
Maybe let's have this set to 3 to make it super obvious it's hidden and the size is unknown.
```suggestion
_PLACEHOLDER: Final[str] = "*" * 3
```

Reply:
i use the same as pydantic